src/main.rs

```diff
@@ -647,9 +656,14 @@ fn main() {
         .bind(&listen_addr, move || {
             Ok(RpcService {
                 work_state: work_state.clone(),
+                base_difficulty
             })
         })
         .expect("Failed to bind server");
+    println!("Configured for the {} network with threshold {:x}", match beta {
+        false => "live",
+        true => "beta"
```

Comment:
Usually you'd have a trailing comma after `"beta"`. Alternatively, you could do `if beta { "beta" } else { "live" }`

Reply:
Yep, always learning!